crates/telio-relay/src/derp/http.rs

```diff
@@ -204,24 +232,38 @@ async fn connect_http<R: AsyncRead + Unpin, W: AsyncWrite + Unpin>(
     writer: &mut W,
     server_keepalives: &DerpKeepaliveConfig,
     host: &str,
+    derp_version: DerpVersion,
 ) -> Result<Vec<u8>, Error> {
-    writer
-        .write_all(
-            format!(
-                "GET /derp HTTP/1.1\r\n\
+    let request = match derp_version {
+        // We try to connect to derp V2
+        DerpVersion::V2 => format!(
+            "GET /v2/derp HTTP/1.1\r\n\
                 Host: {host}\r\n\
```

Comment:
nitpick: Different identation from the `DerpVersion::V1` case, is it on purpose?